operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -170,10 +170,12 @@ private KeyStore loadKeyStoreFile(String ksFilename, String ksPassword) {
      * @return
      * @throws Exception
      */
-    public String submitExport(Instant since, List<String> types, Map<String, String> properties, ExportType exportType)
+    public String submitExport(Instant since, List<String> types, Map<String, String> properties, ExportType exportType,
+        String systemExportVersion)
             throws Exception {
```

Comment:
minor style suggestion
```suggestion
    public String submitExport(Instant since, List<String> types, Map<String, String> properties, ExportType exportType,
            String systemExportVersion) throws Exception {
```

Reply:
I could definitely live with the throws on its own line if you prefer that, I'm just trying to mage sure we double-indent on continuations of method declarations